charts/rancher-thanos/v0.0.2/values.yaml

```diff
@@ -122,7 +122,7 @@ ui:
   replicaCount: 1
   image:
     repository: cnrancher/global-monitoring-ui
-    tag: 0.0.2
+    tag: 0.0.3
```

Comment:
What is it for?

Reply:
it was also using alpine. Updated base image here